admin/src/main/kotlin/dev/cubxity/libs/kdp/admin/repeat.kt

```diff
@@ -0,0 +1,21 @@
+package dev.cubxity.libs.kdp.admin
+
+import dev.cubxity.libs.kdp.perms.botAdmin
+import dev.cubxity.libs.kdp.processing.CommandProcessingPipeline
+
+fun AdminModule.repeat() = AdminModule.repeat {
+    botAdmin = true
+    handler {
+        val times: Int = args["amount"]!!
+        for (i in 0..times) {
```

Comment:
that would run it more than times, use `repeat(times)` also you don't need to call args["command"] every time it loops